src/components/ItaliaTheme/Blocks/HeroImageLeft/StoresButtons.jsx

```diff
@@ -4,21 +4,34 @@ import { Button } from 'design-react-kit/dist/design-react-kit';
 
 import { Icon } from '@italia/components/ItaliaTheme';
 import { UniversalLink } from '@plone/volto/components';
+import { CardReadMore } from 'design-react-kit/dist/design-react-kit';
+import { flattenToAppURL } from '@plone/volto/helpers';
 
 const StoresButtons = ({ data }) => (
   <div className="stores-buttons">
-    {data.playStoreLink && (
-      <Button tag={UniversalLink} href={data.playStoreLink} color="primary">
-        <span>PLAY STORE</span>
-        <Icon icon="fab google-play" />
-      </Button>
-    )}
+    <div className="buttons">
+      {data.playStoreLink && (
+        <Button tag={UniversalLink} href={data.playStoreLink} color="primary">
+          <span>PLAY STORE</span>
+          <Icon icon="fab google-play" />
+        </Button>
+      )}
+
+      {data.appStoreLink && (
+        <Button tag={UniversalLink} href={data.appStoreLink} color="primary">
+          <span>APP STORE</span>
+          <Icon icon="fab apple" />
+        </Button>
+      )}
+    </div>
 
-    {data.appStoreLink && (
-      <Button tag={UniversalLink} href={data.appStoreLink} color="primary">
-        <span>APP STORE</span>
-        <Icon icon="fab apple" />
-      </Button>
+    {data.moreHref && (
+      <CardReadMore
+        tag={UniversalLink}
+        iconName="it-arrow-right"
+        text={data.moreTitle || 'Link ad altro'}
```

Comment:
@SaraBianchi mettiamo la string tradotta. Dovrebbe già essercene una del tipo 'leggi' in altri blocchi, possiamo usare quella